docs: add managing userguide (#161)
ApeWorX/silverback: docs/userguides/deploying.md

In this guide, we are going to show you more details on how to deploy your application to the [Silverback Platform](https://silverback.apeworx.io).

```{note}
You will need to have created a cluster in your workspace first. More information can be found in the [Managing Your Platform](./managing.md).
```

> If you have no bots, we will first have to containerize our Bots and upload them to a container registry that our Cluster is configured to access.

Reviewer: ```suggestion
In order to deploy your bots to the Cluster, we will first have to containerize it and upload them to a container registry that our Cluster can access.
```
